go/vt/topo/tablet.go

```diff
@@ -295,8 +305,10 @@ func (ts *Server) GetTabletsByCell(ctx context.Context, cellAlias string) ([]*Ta
 	if err != nil || len(listResults) == 0 {
 		// Currently the ZooKeeper and Memory topo implementations do not support scans
 		// so we fall back to the more costly method of fetching the tablets one by one.
-		if IsErrType(err, NoImplementation) {
-			return ts.GetTabletsIndividuallyByCell(ctx, cellAlias)
+		// In the etcd case, it is possible that the response is too large. We also fall
```

Comment:
IIRC, ZK has an even smaller limit so I'd avoid mentioning etcd specifically or make it clear that it's an example. If we do mention etcd specifically, it might also be good to add a link or example, e.g.:
```
rpc error: code = ResourceExhausted desc = grpc: trying to send message larger than max (4195197702 vs. 2147483647)
```